netlify.toml

```diff
@@ -0,0 +1,3 @@
+[build]
+  command = "yarn install && yarn build && cd packages/graphiql/example && yarn install"
```

Comment:
might need `yarn install --no-scripts`, to avoid triggering all the `prepublish` scripts